src/dbcp/transform/gridstatus.py

```diff
@@ -390,7 +450,11 @@ def _clean_resource_type(resource_df: pd.DataFrame) -> pd.DataFrame:
 
     unmapped = resource_df["resource_clean"].isna()
     if unmapped.sum() != 0:
-        debug = resource_df[unmapped]["resource"].value_counts(dropna=False)
+        unmapped = resource_df[unmapped]
+        unmapped = unmapped.merge(
+            normalized_projects, on="project_id", how="left", validate="m:1"
+        )
+        debug = unmapped[["region", "resource"]].value_counts(dropna=False).sort_index()
```

Comment:
Merge with `normalized_projects` to get the `region` of projects with unmapped resources.